cmd/atlascmd/project.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package atlascmd
+
+import (
+	"fmt"
+	"os"
+
+	"ariga.io/atlas/schema/schemaspec"
+	"ariga.io/atlas/schema/schemaspec/schemahcl"
+)
+
+const projectFileName = "atlas.hcl"
+
+// projectFile represents an atlas.hcl file.
+type projectFile struct {
+	Envs []*Environment `spec:"env"`
+}
```

Comment:
I'd treat it as an Atlas config rather than an Atlas "project". A config file can store also credentials/tokens to Ariga, AWS, etc.